web/docs/setup/configure.md

```diff
@@ -34,8 +34,8 @@ Each command has its own dedicated set of options. Options are not global and
 only valid for their respective command. Consider this example:
 
 ```bash
-vast --option foo # option applies to command 'vast'
-vast foo --option # option applies to command 'foo'
+tenzir --option foo # option applies to command 'tenzir'
+tenzir foo --option # option applies to command 'foo'
```

Comment:
This should be tenzirctl as tenzir and tenzird have no subcommands.

Reply:
Replacing this with `tenzirctl`. Remember that this commit is just the mechanical renaming. The second pass will be semantic and a lot of things will be completely rewritten.